### PR TITLE
Developed Adopt html

### PR DESCRIPTION
**Pull request by boone824**
Add CSS to project

### CODE REVIEW

File: blog.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+          <li class="links"><a href="index.html">Home</a></li>
+          <li class="links"><a href="dogs.html">Dogs</a></li>
+          <li class="links"><a href="blog.html">Blog</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+        <div class="row">
+          <div class="col">
+            <img src="images/blog-1.jpg" alt="Picture of dog on a cliff">
```
Comment (jimmynono) [inline blog.html:27]: good alt tag!

File: dogs.html [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+      <nav class="style-nav">
+        <ul>
+          <li class="links"><a href="index.html">Home</a></li>
+          <li class="links"><a href="dogs.html">Dogs</a></li>
+          <li class="links"><a href="blog.html">Blog</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="row">
```
Comment (jimmynono) [inline dogs.html:25]: These rows might not be the best method. It works fine for now, but what if you want to show 4 dog tiles wide on desktop, 3 on a tablet and 1 wide on a cell phone? Think of how you could leverage css grid or flexbox to lay these tiles out

File: index.html [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+        </div>
+      <nav class="style-nav">
+        <ul>
+          <li class="links"><a href="index.html">Home</a></li>
+          <li class="links"><a href="dogs.html">Dogs</a></li>
+          <li class="links"><a href="blog.html">Blog</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
```
Comment (jimmynono) [inline index.html:24]: nice use of main!

File: stylesheet/styles.css [hunk 4]
```diff
@@ -0,0 +1,114 @@
+* {
+  box-sizing: border-box;
+}
+body {
+  margin-top: 0px;
+  margin-bottom: 0px;
+  margin-left: 0px;
+  margin-right: 0px;
```
Comment (jimmynono) [inline stylesheet/styles.css:8]: you could combine this to be margin: 0;

File: stylesheet/styles.css [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+}
+header {
+  background-color: black;
+  color: white;
+  padding-bottom: 1rem;
+}
+img {
```
Comment (jimmynono) [inline stylesheet/styles.css:15]: be careful with giving width 100% to ALL images, they might not all need this.

File: stylesheet/styles.css [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+  background-color: #093c60;
+  color: white;
+}
+.home-intro {
+  width: 25%;
+  float: left;
+}
+.intro-content {
+  width: 75%;
+  float: right;
```
Comment (jimmynono) [inline stylesheet/styles.css:36]: Don't use floats for layout. Use flexbox or grid for this.

File: stylesheet/styles.css [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+}
+ .style-nav ul li a {
+   text-decoration: none;
+   color: grey;
+   text-align: center;
+   display: block;
+   text-transform: uppercase;
+   padding: 1rem;
+   font-size: 1rem;
+   float: left;
```
Comment (jimmynono) [inline stylesheet/styles.css:50]: No floats please, you can lay out these in a line (hint hint)

File: stylesheet/styles.css [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+  text-decoration: none;
+  text-align: center;
+}
+.col {
+  margin-top: 1rem;
+  text-align: center;
+  margin-bottom: 1rem;
+
+}
+.row {
```
Comment (jimmynono) [inline stylesheet/styles.css:93]: Your row/col system doesn't seem to work. The tiles are full width on my laptop. You'll want to use flexbox or grid to lay them out. A single wrapper around them might be helpful. Especially as we might not want to have 4 tiles wide on laptop, 3 on tablet and 1 on mobile. Flexbox/grid will be helpful

**Comment by boone824** [comment]
Ok...I will change html to flexbox..thanks for the help..

**Pull request closed without merging**